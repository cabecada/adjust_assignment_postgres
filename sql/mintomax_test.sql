create extension mintomax;
select array_to_string(mintomax(x), '->') from generate_series(1, 5) x;
select array_to_string(mintomax(x), '::') from generate_series(1, 5) x;
