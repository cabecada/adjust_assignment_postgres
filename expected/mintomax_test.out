create extension mintomax;
select array_to_string(mintomax(x), '->') from generate_series(1, 5) x;
 array_to_string 
-----------------
 1->5
(1 row)

select array_to_string(mintomax(x), '::') from generate_series(1, 5) x;
 array_to_string 
-----------------
 1::5
(1 row)

